web/templates/files/details.html

```diff
@@ -16,12 +16,21 @@ <h4 class="probable-names">Probable Name(s): {{file.probable_names|join(', ')}}<
                             </form>
                         </span>
                     {% endfor %}
+
                     <a href="#" id="add-sharing-group">+</a>
+                    <span class="filter-btn">
+                        <form method="POST" action="/analyses/{{analysis._id}}/reviewed">
```

Comment:
We can't do this because `analysis` will not be defined when looking at the object's page.